optuna/integration/allennlp/_pruner.py

```diff
@@ -227,4 +245,16 @@ def on_epoch(
 
         self._trial.report(float(value), epoch)
         if self._trial.should_prune():
+            trainer._metric_tracker.should_stop_early = lambda: True  # type: ignore
```

Comment:
To stop a training loop, I replace an AllenNLP built-in stopper with a function that always returns `True`.